doc/manual/source/interaction/geoserver_ui.rst

```diff
@@ -174,40 +157,40 @@ It is then possible to clone this repository::
 
     $ geogig clone http://localhost:8080/geoserver/geogig/repos/geogig_dir_repo geogig_dir_repo
 
-Your clone will be configured with the geoserver repository as a remote. This configuration is stored in .geogig/config in
-your clone::
+Your clone will be configured with the GeoServer repository as a remote. This configuration is stored in .geogig/config in your clone::
 
     [remote\origin]
     url = http://localhost:9090/geoserver/geogig/repos/geogig_dir_repo
     fetch = +refs/heads/*:refs/remotes/origin/*
-    
+
     [branches\master]
     remote = origin
     merge = refs/heads/master
 
-It is now possible to push and pull from this remote repository. You can verify this works by testing with the freshly
-cloned repo::
+It is now possible to push and pull from this remote repository. You can verify this works by testing with the freshly cloned repository.
+
+::
 
     $ geogig push origin
     Nothing to push.
-    
+
     $ geogig pull origin
     100%
     Already up to date.
 
-Automated Repository Synchronization
+Automated repository synchronization
 ------------------------------------
 
-Repositories configured by GeoServer can be configured with remotes and Automated Repository Syncrhonization. TODO
+Repositories configured by GeoServer can be configured with remotes and Automated Repository Synchronization. TODO
 
 .. _current-limitations:
 
-Current Limitations
+Current limitations
 -------------------
 
-When using Directory backed GeoGig repositories, the default underlying object database (berkeley db) is single user. While
-the repository is being exposed over the network by either the stand-alone server or by GeoServer, you will not be able to
-access the repo from the command line interface. The error is pretty clear about whats going on.
+When using Directory-backed GeoGig repositories, the default underlying object database (BerkeleyDB) is single-user. While the repository is being exposed over the network by either the stand-alone server or by GeoServer, you will not be able to access the repository from the command line interface. The error is pretty clear about whats going on.
+
+::
```

Comment:
The exception text below needs to be indented if you are using the literal block marker `::` here